tests/update/zypper_up.pm

```diff
@@ -1,23 +1,26 @@
 use strict;
 use base "openQAcoretest";
 use testapi;
+use utils;
 
 sub run {
-    assert_screen "openqa-desktop", 500;
+    wait_for_desktop;
     send_key "ctrl-alt-f2";
     assert_screen "inst-console";
     type_string "root\n";
-    assert_screen "password-prompt", 10;
+    assert_screen "password-prompt";
     type_string "1\n";
```

Comment:
There is "select_console" but I first have to setup all consoles in lib/susedistribution.pm or clean it up more. Remember, we are in os-autoinst-distri-openQA, not os-autoinst-distri-opensuse.

The password "1" is the password that has been used in the pre-generated 13.2 image that has an already installed openQA instance included.